public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -175,7 +190,7 @@ function getCollapsedInfo(query: LokiQuery, queryType: LokiQueryType, maxLines:
 
   if (!isLogQuery) {
     if (query.step) {
-      items.push(`Step: ${query.step}`);
+      items.push(`Step: ${isValidStep ? query.step : 'Invalid step'}`);
```

Comment:
I don't like a lot reading "Step: Invalid step" (2x step). Maybe:

```suggestion
      items.push(`Step: ${isValidStep ? query.step : 'Invalid value'}`);
```

Reply:
Good point! Updated.